Source/DiabloUI/settingsmenu.cpp

```diff
@@ -73,16 +73,24 @@ void GoBackOneMenuLevel()
 	shownMenu = ShownMenuType::Settings;
 }
 
+void UpdateDescription(OptionEntryBase *pOption)
+{
+	auto paragraphs = WordWrapString(pOption->GetDescription(), rectDescription.size.width, GameFont12, 1);
+	strncpy(optionDescription, paragraphs.c_str(), sizeof(optionDescription));

```

Comment:
GetDescription returns a translated string of potentially multibyte utf8 codepoints, might as well use CopyUtf8 here to be safe.
```suggestion
	CopyUtf8(optionDescription, paragraphs, sizeof(optionDescription));
```

Reply:
Fixed 🙂